src/System.Runtime.WindowsRuntime/src/System/IO/StreamOperationsImplementation.cs

```diff
@@ -11,6 +13,8 @@
 using Windows.Foundation;
 using Windows.Storage.Streams;
 
+using SREMemoryStream = System_Runtime_Extensions::System.IO.MemoryStream;
```

Comment:
we should probably add a comment in this hack on to why we need to use a different name, just for future reference.